cmd/cluster/aws/create.go

```diff
@@ -504,3 +499,46 @@ func validateAWSOptions(ctx context.Context, opts *core.CreateOptions, awsOpts *
 
 	return nil
 }
+
+func validateMultiArch(ctx context.Context, o *RawCreateOptions, opts *core.CreateOptions) error {
+	var validMultiArchImage bool
```

Comment:
```suggestion
	validMultiArchImage := false
```
nit: make it more explicit